server/plugin/command.go

```diff
@@ -265,6 +271,50 @@ func (p *Plugin) handleSubscriptionsList(_ *plugin.Context, args *model.CommandA
 	return txt
 }
 
+func (p *Plugin) getWebhookListForRepoOrOrg(githubClient *github.Client, repo, owner string, ctx context.Context) (bool, error) {
+	isWebhook := false
```

Comment:
`isWebhook` does seem intention revealing to me. Maybe
```suggestion
	var found bool
```